website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -0,0 +1,120 @@
+
+To use a different controller, see [the ingress controller documentation](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/).
+
+To learn more about enabling this feature on your cluster, see the [Ingress prerequisites](https://kubernetes.io/docs/concepts/services-networking/ingress/#prerequisites) on the official kubernetes documentation.
+
+## Installing the Operator Lifecycle Manager(OLM)
+The Operator Lifecycle Manager(OLM) is a component of the Operator Framework, an open source toolkit to manage Kubernetes native applications, called Operators, in a streamlined and scalable way.[(Source)](https://olm.operatorframework.io/)
+
+[//]: # (Move this section to Architecture > Service Binding or create a new Operators doc)
+What are Operators?
+>The Operator pattern aims to capture the key aim of a human operator who is managing a service or set of services. Human operators who look after specific applications and services have deep knowledge of how the system ought to behave, how to deploy it, and how to react if there are problems.
+>
+>People who run workloads on Kubernetes often like to use automation to take care of repeatable tasks. The Operator pattern captures how you can write code to automate a task beyond what Kubernetes itself provides.
+> [(Source)](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/#motivation)
+[//]: # (Move until here)
+
+To install an Operator, we will first need to install OLM [(Operator Lifecycle Manager)](https://olm.operatorframework.io/) on the cluster.
+```shell
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.3/install.sh | bash -s v0.18.3
+```
+Running the script will take some time to install all the necessary resources in the Kubernetes cluster including the `OperatorGroup` resource,....
+
+Note: Check the OLM [release page](https://github.com/operator-framework/operator-lifecycle-manager/releases/) to use the latest version.
+
+## Installing the Service Binding Operator
+odo uses [Service Binding Operator](https://operatorhub.io/operator/service-binding-operator) to provide the `odo link` feature which helps to connect an odo component to a service or another component.
+
+Operators can be installed in a specific namespace or across the cluster(i.e. in all the namespaces).
+```shell
+kubectl create -f https://operatorhub.io/install/service-binding-operator.yaml
+```
+Running the command will create the necessary resource in the `operators` namespace.
+
+If you want to access this resource from other namespaces as well, add your target namespace to `.spec.targetNamespaces` list.
+
+See [Verifying the Operator installation](#verifying-the-operator-installation) to ensure that the Operator was installed successfully.
+
+## Installing an Operator
```

Comment:
You can add a little sentence to introduce what will be done in the following steps, smth similar to:

```
To install an operator from the OperatorHub website:
```